### PR TITLE
Fix example endpoint code ExecutionParameters.java to work with SageM…

### PR DESCRIPTION
**Pull request by ericangelokim**
…aker batch.

Before or while filing an issue please feel free to join our [ slack channel](https://join.slack.com/t/mms-awslabs/shared_invite/enQtNDk4MTgzNDc5NzE4LTBkYTAwMjBjMTVmZTdkODRmYTZkNjdjZGYxZDI0ODhiZDdlM2Y0ZGJiZTczMGY3Njc4MmM3OTQ0OWI2ZDMyNGQ) to get in touch with development team, ask questions, find out what's cooking and more!

## Description of changes:
The endpoint was returning a json of unicode, but MaxConcurrentTransforms and MaxPayloadInMB fields are expected to return int by batch.

## Testing done:
Used endpoint jar in XGBoost: https://github.com/aws/sagemaker-xgboost-container/pull/41

**To run CI tests on your changes refer [README.md](https://github.com/awslabs/mxnet-model-server/blob/master/ci/README.md)**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by vdantu** [comment]
@alexwong @alexgl-github : this PR is very useful to MMS customers. It would be good to have this in our plugins. Would you be able to work with @ericangelokim to get this in.

File: plugins/endpoints/src/main/java/software/amazon/ai/mms/plugins/endpoint/ExecutionParameters.java [hunk 1]
```diff
@@ -33,4 +51,46 @@ public void doGet(Request req, Response rsp, Context ctx) throws IOException {
                                 .toJson(r)
                                 .getBytes(StandardCharsets.UTF_8));
     }
+
+    /** Response for Model server endpoint */
+    public static class SagemakerXgboostResponse {
```
Comment (vdantu) [inline plugins/endpoints/src/main/java/software/amazon/ai/mms/plugins/endpoint/ExecutionParameters.java]: Could we rename this class to be made generic. Call it "ExecutionParametersResponse" or some such.

File: plugins/endpoints/src/main/java/software/amazon/ai/mms/plugins/endpoint/ExecutionParameters.java [hunk 2]
```diff
@@ -12,19 +12,37 @@
 import software.amazon.ai.mms.servingsdk.http.Request;
 import software.amazon.ai.mms.servingsdk.http.Response;
 
+// The modified endpoint source code for the jar used in this container.
+// You can create this endpoint by moving it by cloning the MMS repo:
+// > git clone https://github.com/awslabs/mxnet-model-server.git
+//
+// Copy this file into plugins/endpoints/src/main/java/software/amazon/ai/mms/plugins/endpoints/
```
Comment (vdantu) [inline plugins/endpoints/src/main/java/software/amazon/ai/mms/plugins/endpoint/ExecutionParameters.java]: You don't need these comments anymore. I think its documented in the docs.

File: plugins/endpoints/src/main/java/software/amazon/ai/mms/plugins/endpoint/ExecutionParameters.java [hunk 3]
```diff
@@ -33,4 +35,46 @@ public void doGet(Request req, Response rsp, Context ctx) throws IOException {
...
+        @SerializedName("MaxConcurrentTransforms")
+        private int maxConcurrentTransforms;
+
+        @SerializedName("BatchStrategy")
+        private String batchStrategy;
+
+        @SerializedName("MaxPayloadInMB")
+        private int maxPayloadInMB;
+
+        public SagemakerXgboostResponse() {
```
Comment (vdantu) [inline plugins/endpoints/src/main/java/software/amazon/ai/mms/plugins/endpoint/ExecutionParameters.java]: Could you rename this as well? I am not sure how this compiled.

File: plugins/endpoints/src/main/java/software/amazon/ai/mms/plugins/endpoint/ExecutionParameters.java [hunk 4]
```diff
@@ -33,4 +35,46 @@ public void doGet(Request req, Response rsp, Context ctx) throws IOException {
...
+        private int maxConcurrentTransforms;
+
+        @SerializedName("BatchStrategy")
+        private String batchStrategy;
+
+        @SerializedName("MaxPayloadInMB")
+        private int maxPayloadInMB;
+
+        public ExecutionParametersResponse() {
+            maxConcurrentTransforms = 4;
```
Comment (vdantu) [inline plugins/endpoints/src/main/java/software/amazon/ai/mms/plugins/endpoint/ExecutionParameters.java]: NIT: Nothing to change immediately. I don't see a value for the constructor here. Seems like everything is being populated :).

File: plugins/endpoints/src/main/java/software/amazon/ai/mms/plugins/endpoint/ExecutionParameters.java [hunk 5]
```diff
@@ -17,14 +17,16 @@
...
-        HashMap<String, String> r = new HashMap<>();
-        r.put("MAX_CONCURRENT_TRANSFORMS", prop.getProperty("NUM_WORKERS", "1"));
-        r.put("BATCH_STRATEGY", "SINGLE_RECORD");
-        r.put("MAX_PAYLOAD_IN_MB", prop.getProperty("max_request_size"));
-        r.put("BATCH", "true");
+        // 6 * 1024 * 1024
+        int maxRequestSize = Integer.parseInt(prop.getProperty("max_request_size", "6291456"));
+        ExecutionParametersResponse r = new ExecutionParametersResponse();
+        r.setMaxConcurrentTransforms(Integer.parseInt(prop.getProperty("NUM_WORKERS", "1")));
+        r.setBatchStrategy("MULTI_RECORD");
```
Comment (vdantu) [inline plugins/endpoints/src/main/java/software/amazon/ai/mms/plugins/endpoint/ExecutionParameters.java]: Is this always MULTI_RECORD?

Comment (ericangelokim) [reply plugins/endpoints/src/main/java/software/amazon/ai/mms/plugins/endpoint/ExecutionParameters.java]: To my knowledge yes

**Comment by vdantu** [comment]
@alexwong : do you have any comments on this PR? I think this is merge-able, unless you feel otherwise :)

**Comment by vdantu** [comment]
Merging this . Testing was done by @ericangelokim .

**Pull request merged**